cumulusci/tasks/github.py

```diff
@@ -207,3 +210,51 @@ def _run_task(self):
                 )
 
                 self.logger.info('Merge conflict on branch {}: created pull request #{}'.format(branch.name, pull.number))
+
+
+class CommitApexDocs(BaseGithubTask):
+
+    task_options = {
+        'branch': {
+            'description': 'branch name',
+            'required': True,
+        },
+        'dir_local': {
+            'description': 'local dir of ApexDocs (contains index.html)',
+            'required': True,
+        },
+        'dir_repo': {
```

Comment:
Thinking about this more, it really has nothing to do with remote vs local either, it is all about where in the repo you want it to be (repo is same whether local or remote). I'd like to leave it as `dir_repo` for clarity.